cfn/deploy_dev_ec2.yaml

```diff
@@ -0,0 +1,148 @@
+Parameters:
+  InstanceTypeParameter:
+    Type: String
+    Default: "m6i.large"
+    AllowedValues:
+      - t2.micro
+      - m5.small
+      - m6i.large
+    Description: Default is t2.micro.
+  KeyIDParameter:
+    Type: String
+    Default: "globalrx_sshkey2"
```

Comment:
There are multiple places referring "globalrx" still. Should we change them to searchrx?

Reply:
ya, good point. I'll updated and create a new PR